os/cluster-architectures.md

```diff
@@ -60,22 +49,43 @@ coreos:
         ExecStart=/usr/bin/systemctl enable docker-tcp.socket
 ```
 
+This file is used to provision your local CoreOS machine on it's first boot. It enables the docker sets up and enables the docker socket in systemd.
+
+Using the Container Linux Config Transpiler, or `ct`, ([download][ct-getting-started]) convert the above yaml into an [ignition][ignition-getting-started]. Once you have the ignition configuration file, pass it to your provider ([complete list of supported ignition platforms][ignition-supported]).
```

Comment:
... the above yaml into an Ignition configuration file. Once you have the config file, ...

('Ignition' looks to be capitalized everywhere in the CoreOS docs. Is it a product? strange.)